### ISSUE TITLE
OpenSSL 3.2.0-alpha1: Unintended use of a perl 5.14 feature in 50-win-hybridcrt.conf

### ISSUE THREAD

**Issue by rainerjung** [openssl/openssl#22016]
File 50-win-hybridcrt.conf uses the regexp modifier "r" introduced in Perl 5.14. If things haven't changed, OpenSSL is expected to be buildable using Perl 5.11. I suggest the following trivial patch:

```
--- 50-win-hybridcrt.conf	2023-09-07 19:05:57.909701833 +0200
+++ 50-win-hybridcrt.conf	2023-09-07 22:54:26.948999962 +0200
@@ -11,7 +11,8 @@
 sub remove_from_flags {
     my ($toRemove, $flags) = @_;
     
-    return $flags =~ s/$toRemove//r;
+    $flags =~ s/$toRemove//;
+    return $flags;
 }
 
 my %targets = (
```

The patch is needed eg. to build on RHEL 6 or SLES 11 and similar old systems.

Thanks and regards,

Rainer

**Comment by rsbeckerca** [comment]
@rainerjung Do you need assistance with a pull request?

**Comment by rainerjung** [comment]
It is a trivia change, so hopefully  a pull request is not needed.

**Comment by h-vetinari** [comment]
A pull request is always needed, even for trivial changes (no-one commits to master directly).

**Comment by rainerjung** [comment]
Sure, but a pull request for a trivial change doesn't have to be done by the reporter. Nevertheless: PR #22019.

**Comment by mattcaswell** [comment]
This was fixed by #22109. Closing.
